### DISCUSSION TITLE
Using Yao.jl in my paper "Using the Julia framework to teach quantum entanglement"

### DISCUSSION THREAD

**Discussion by BoltzmannEntropy** [QuantumBFS/Yao.jl#475] [Show and tell]
https://arxiv.org/pdf/2302.12889.pdf

Best,

**Comment by Elphikaus** [comment]
Looks really interesting. I will enjoy reading this.

**Comment by Elphikaus** [comment]
Though this could be a good candidate for "Discussions"?
